README.md

```diff
@@ -50,6 +52,20 @@ pipenv run python -m spacy download en_core_web_sm
 pipenv run python -m spacy download en_core_web_md
 ```
 
+Alternatively, run a Docker container with all necessary dependencies and Spacy models installed.
+
+Run the Docker container for Mac:
+
+```bash
+sh docker_run_mac.sh
+```
+
+Run the Docker container for Windows:
+
+```
+docker_run_win.bat
+```
+
```

Comment:
I think it would be nice to include the steps to run the tool in Docker under its own subheader. This way users can quickly identify that there is an available docker image that they could use. Additionally, I think that you should include more information here about what type of command-line arguments the scripts are using. This is useful for users interested in specifics. Also, the file names used in these commands are not actually present. There is a discrepancy in the file names and their paths, the commands do not specify that the files are in the `scripts` folder and thus they might not be found. Please make sure to test these more. Lastly, the readme only includes steps for mac and windows (**sad Linux noises**). It would be great if you can the steps for popular Linux distribution like Ubuntu.

Reply:
Good point. About the operating system, I feel the Mac command should just work as well on a Linux machine? This should be further tested out and the instruction in the documentation will need to be updated.